1.0/docs/devguide/tests.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: default
+type: guide
+shortname: Docs
+title: Unit tests
+subtitle: Developer guide
+---
+
+<link rel="import" href="../../../0.5/components/google-youtube/google-youtube.html">
+
+{% include toc.html %}
+
+Use Web Component Tester (`wct`) to unit test your Polymer elements. 
+
+`wct` is an end-to-end testing environment built by the Polymer core team. It
+enables you to test your elements locally, against all of your installed
+browsers, or remotely, via Sauce Labs. `wct` is composed of the following
+tools:
```

Comment:
:heavy_check_mark: "composed of" --> "built on top of popular third-party tools"
:heavy_check_mark: added clarification that the guide uses `wct` CLI to refer to WCT
:heavy_check_mark: pinged @azakus via Hangouts to make sure that the definition of WCT is OK